File: 03multimedia/01buzzer.js
var wpi = require('wiring-pi');
var sleep = require('sleep');

var pin = 18

wpi.wiringPiSetupGpio()
wpi.softToneCreate(pin)
while (true) {
	wpi.softToneWrite(pin, 392)
	sleep.msleep(500);
	wpi.softToneWrite(pin, 523)
	sleep.msleep(500);
}

process.on('exit', function() {
	wpi.softToneWrite(pin, 0);
});

File: 03multimedia/01buzzer.py
import wiringpi  
from time import sleep  
PIN_BUZ = 18

wiringpi.wiringPiSetupGpio()  
  
try:
    wiringpi.softToneCreate(PIN_BUZ)
    while True:
        wiringpi.softToneWrite(PIN_BUZ, 392)
        sleep(0.5)
        wiringpi.softToneWrite(PIN_BUZ, 523)
        sleep(0.5)
finally:
    wiringpi.pinMode(PIN_BUZ, 0)
